gridfs/asynchronous/grid_file.py

```diff
@@ -1484,6 +1486,17 @@ def __init__(
     _file: Any
     _chunk_iter: Any
 
+    async def __anext__(self) -> bytes:
+        return super().__next__()
```

Comment:
This is incorrect. We can't call `super().__next__()` because that does blocking I/O.

Reply:
Hmm, good catch. We don't have an async equivalent here unless we write one ourselves.

Reply:
IOBase implements next using readline:
> [IOBase](https://docs.python.org/3/library/io.html#io.IOBase) (and its subclasses) supports the iterator protocol, meaning that an [IOBase](https://docs.python.org/3/library/io.html#io.IOBase) object can be iterated over yielding the lines in a stream. Lines are defined slightly differently depending on whether the stream is a binary stream (yielding bytes), or a text stream (yielding character strings). See [readline()](https://docs.python.org/3/library/io.html#io.IOBase.readline) below.

https://docs.python.org/3/library/io.html#io.IOBase

Reply:
There isn't an `asyncio` version of `readline`, so we'd need to write our own. The canonical way to do so appears to be with threads (https://stackoverflow.com/questions/34699948/does-asyncio-supports-asynchronous-i-o-for-file-operations), at which point I question if the performance gained by not blocking the loop is more than the cost of thread overhead.